src/main/resources/openapi/PrintAPIs.yaml

```diff
@@ -875,7 +875,7 @@ components:
         - applicationReference
         - electoralRollNumber
         - certificateLanguage
-        - supportingInfoFormat
+        - supportingInformationFormat
```

Comment:
Isnt `supportingInformationFormat` optional in dto that you created? How does that work